src/dev/src/app/timeline/timeline.demo.routing.ts

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (c) 2016-2019 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+import { ModuleWithProviders } from '@angular/core';
+import { RouterModule, Routes } from '@angular/router';
+
+import { TimelineTypesDemo } from './timeline-types';
+import { TimelineDemo } from './timeline.demo';
+
+const ROUTES: Routes = [
+  {
+    path: '',
+    component: TimelineDemo,
+    children: [
+      { path: '', redirectTo: 'timeline-types', pathMatch: 'full' },
+      { path: 'timeline-types', component: TimelineTypesDemo },
```

Comment:
This can be the blank path route and remove the redirect above. We don't need two routes.

Reply:
Great suggestion, removed all routes make it one component only. Followed another example before and thought there might be multiple demo pages required.